tools/account_diff_table.py

```
@@ -0,0 +1,212 @@
+
+def load_rows(path: Path) -> list[dict[str, str]]:
+    suffix = path.suffix.lower()
+    if suffix == ".csv":
+        with path.open("r", encoding="utf-8-sig", newline="") as file_obj:
+            return list(csv.DictReader(file_obj))
+    if suffix == ".json":
+        with path.open("r", encoding="utf-8") as file_obj:
+            data = json.load(file_obj)
+        if isinstance(data, list):
+            return [row for row in data if isinstance(row, dict)]
+        raise ValueError(f"JSON file must contain a list of objects: {path}")
```

Ruff (0.15.10)

[warning] Avoid specifying long messages outside the exception class

(TRY003)